db/disk.go

```diff
@@ -18,14 +18,19 @@
 
 package db
 
-import "database/sql"
+import (
+	"database/sql"
+	"fmt"
+)
 
 type Disk struct {
-	Machine    *Machine
+	id	   int
```

Comment:
It looks like these fields haven't been indented properly... did you run `go fmt` on all of this code before committing/pushing?